### ISSUE TITLE
Writes build pid to default configuration (instead of $$ for CREATE FUNCTION queries)

### ISSUE THREAD

**Issue by nthykier** [darold/ora2pg#397]
Hi,

FYI, you might be interested in the following Debian bug (https://bugs.debian.org/861443), where it was reported that ora2pg version 18.0 generates a default configuration with the "build PID" in it.  Looking through the proposed patch for fixing it, it seems that the "$$" in the psql queries are interpreted as the perl variable for "current PID".

```
+@@ -288,7 +290,7 @@ CONTEXT_AS_TRGM		0
+ # To automatically update the column when a modification in the title column
+ # appears, Ora2Pg adds the following trigger:
+ #
+-# CREATE FUNCTION tsv_t_document_title() RETURNS trigger AS $$
++# CREATE FUNCTION tsv_t_document_title() RETURNS trigger AS $T
+ # BEGIN
+ #        IF TG_OP = 'INSERT' OR new.title != old.title THEN
+ #                new.tsv_title :=
+@@ -296,7 +298,7 @@ CONTEXT_AS_TRGM		0
+ #        END IF;
+ #        return new;
+ # END
+-# $$ LANGUAGE plpgsql;
++# $T LANGUAGE plpgsql;
+ # CREATE TRIGGER trig_tsv_t_document_title BEFORE INSERT OR UPDATE
+ #  ON t_document
+ #  FOR EACH ROW EXECUTE PROCEDURE tsv_t_document_title();
+@@ -325,9 +327,9 @@ FTS_INDEX_ONLY	1
+ #
+ #	CREATE OR REPLACE FUNCTION unaccent_immutable(text)
+ #	RETURNS text AS
+-#	$$
++#	$T
+ #		SELECT public.unaccent('public.unaccent', $1)
+-#	$$  LANGUAGE sql IMMUTABLE
++#	$T  LANGUAGE sql IMMUTABLE
+ #	    COST 1;
+ #
+ # indexes are exported as follow:
+@@ -347,9 +349,9 @@ USE_UNACCENT		0
+ #
+ #      CREATE OR REPLACE FUNCTION unaccent_immutable(text)
+ #      RETURNS text AS
+-#      $$
++#      $T
+ #          SELECT lower(public.unaccent('public.unaccent', $1));
+-#      $$ LANGUAGE sql IMMUTABLE;
++#      $T LANGUAGE sql IMMUTABLE;
+ #
+ USE_LOWER_UNACCENT	0
+ 
```

(The proposed patch is otherwise wrong because the submitter seems to have misunderstood the intend of the '$$' symbol)

**Comment by nthykier** [comment]
Here is the results from the reproducible-builds tests from Debian:

https://tests.reproducible-builds.org/debian/rb-pkg/experimental/amd64/diffoscope-results/ora2pg.html

(scroll to "./etc/ora2pg/ora2pg.conf" entry)

**Comment by nthykier** [comment]
Apologies, I missed the fact that this was fixed in 18.1. :)

**Comment by darold** [comment]
Hi Niels,

That's correct, commits 8f2901d and 0573bd1 fix some character issues in configuration file.

Best regards,
